xchange-stream-core/pom.xml

```diff
@@ -15,6 +15,12 @@
             <artifactId>service-core</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        
+        <dependency>
```

Comment:
this should not be in core as dependency. These services are only for exchange implementations. Why it is here?

Reply:
POM file was pushed by mistake. Please ignore it.